web/src/i18n/locales/ja.json

```
@@ -5,11 +5,11 @@
         "title": "404：ページが見つかりません"
     },
     "about": {
-        "purpose": "登録者の情報は、イベントの企画者が女性スピーカーを簡単に見つけられるようにするため、また、ご登録いただいた方々が繋がって新しいコミュニティーを作るために使われます。ご注意 : Eメールアドレスを除き登録された情報はWebサイト上に公開されます。公開を望まない情報は共有しないでください。",
+        "purpose": "登録者の情報は、イベントの企画者が女性のスピーカーを簡単に見つけられるようにする目的に使用されます。ご登録いただいた方々がつながって新しいコミュニティーを作るために活用されます。ご注意 : Eメールアドレスを除き登録された情報はWebサイト上に公開されます。公開を望まない情報は共有しないでください。",
```

Please review the comments: https://github.com/WWCodeTokyo/speak-her-db/pull/118#discussion_r485634258

---

OK

---

@shokokb I'm putting the comment here for your convenience. Please tell me what you think!:
I would add `または、` like `または、ご登録いただいた方々がつながって` before the second sentence. What do you think? I feel it's creating a better connection between the sentences. Also if you think it's better we can remove `Web` from `Webサイト` in this sentence too.

---

I already linked the prior comment (but didn't copy paste), and it appears Shoko has okayed it. Can you please use [suggested changes](https://thenextweb.com/dd/2018/10/16/github-launches-suggested-changes-to-make-it-easier-to-collaborate-within-pull-requests/) so I know how to appropriately update the branch?

---

Actually let me try it... Does this look okay to everyone?
```suggestion
        "purpose": "登録者の情報は、イベントの企画者が女性のスピーカーを簡単に見つけられるようにする目的に使用されます。または、ご登録いただいた方々がつながって新しいコミュニティーを作るために活用されます。ご注意 : Eメールアドレスを除き登録された情報はWebサイト上に公開されます。公開を望まない情報は共有しないでください。",
```

---

@ann-kilzer perfect!

---

```suggestion
        "purpose": "登録者の情報は、イベントの企画者が女性スピーカーを簡単に見つけられるようにする目的に使用されます。または、ご登録いただいた方々がつながって新しいコミュニティーを作るために活用されます。ご注意 : Eメールアドレスを除き登録された情報はWebサイト上に公開されます。公開を望まない情報は共有しないでください。",
```